src/v/pandaproxy/rest/configuration.cc

```diff
@@ -47,6 +50,12 @@ configuration::configuration()
       "api_doc_dir",
       "API doc directory",
       config::required::no,
-      "/usr/share/redpanda/proxy-api-doc") {}
+      "/usr/share/redpanda/proxy-api-doc")
+  , consumer_instance_timeout(
+      *this,
+      "consumer_instance_timeout_ms",
+      "How long to wait for an idle consumer before removing it",
+      config::required::no,
+      std::chrono::minutes{5}) {}
```

Comment:
nit: can use `std::chrono_literals`

Reply:
Yeah. I quite like spelling it out for some reason.